### DISCUSSION TITLE
About the meaning of the CI comment regarding screenshot test

### DISCUSSION THREAD

**Discussion by sanao1006** [android/nowinandroid#1197] [Q&A] [answered]
Hello!

https://github.com/android/nowinandroid/blob/96b554d6a2240103290cd4c5406e9e596d2e7ea1/.github/workflows/Build.yaml#L102-L104

> TODO: Ignore screenshots.

I found a TODO comment on line 103 of [`Build.yml`](https://github.com/android/nowinandroid/blob/main/.github/workflows/Build.yaml). But I don't understand the meaning.
In what cases should we ignore screenshots?

What problem should I solve to make this TODO disappear?

@JoseAlcerreca @SimonMarquis,  Someone please tell me

**Comment by JoseAlcerreca** [comment, marked as answer]
Roborazzi tests are run when running `./gradlew test` but they always pass. To optimize a little, ideally we would ignore them. But it's not a huge deal really.

**Reply by sanao1006** [reply]
Thanks!
